uninstall_auth_proxy.yml

```diff
@@ -0,0 +1,23 @@
+---
+# This playbook performs service cleanup for auth_proxy.
+#
+# Note: cleanup is not expected to fail, so we set ignore_errors to yes here
+
+- hosts: netplugin-master
+  become: true
+  tasks:
+    - include_vars: roles/{{ item }}/defaults/main.yml
+      with_items:
+        - "contiv_network"
+        - "contiv_storage"
+        - "swarm"
+        - "kubernetes"
+        - "ucp"
+        - "docker"
+        - "etcd"
+        - "auth_proxy"
```

Comment:
Ideally no. We have some cross references of defaults between docker, etcd and contiv. auth_proxy does not have any dependencies. I will remove the references to contiv_storage and ucp.